neuro_config_client/entities.py

```diff
@@ -65,8 +65,12 @@ class NodePoolOptions:
     available_cpu: float
     memory: int
     available_memory: int
-    gpu: int | None = None
-    gpu_model: str | None = None
+    nvidia_gpu: int | None = None
```

Comment:
This corresponds to NodePoolTemplate on backend, it doesn't support different gpu models